sisl/io/sile.py

```diff
@@ -332,11 +364,11 @@ def base_file(self):
         """ File of the current `Sile` """
         return basename(self._file)
 
-    def dir_file(self, filename=None):
+    def dir_file(self, filename=None, extra_dirs=""):
```

Comment:
Rename `extra_dirs` to `filename_base` to clarify that this is inserted just before `filename`. I.e. it isn't totally clear from the naming convention that `extra_dirs` goes between `_directory` and `filename`

Reply:
I also thought this, but doesn't `filename_base` sound like I will do `filename_base/filename` and not `_directory/filename_base/filename`?

By the way this is just for people that keep their results in an extra folder, so they have this structure:
_ mydir
......|
......| - RUN.fdf
......| - C.psf
......| - results
...............| - graphene.bands
...............| - other files...

Maybe this "results folder" (or filename_base as you called it) could be a kwarg and attribute of `fdfSileSiesta`? What do you think, is it common enough? Or at least there could be a way to specify it when you do `fdf_sile.read_grid("VT")`, for example.

Reply:
Hmm, I don't particularly like this.  
I agree my suggestion is also ambiguous.  
In siesta you can just do `SystemLabel results/graphene` and everything will be placed in the results folder. Then `fdfSile` will automatically handle this abstraction. So I don't think it is necessary.

Optionally one can initialize the `fdfSile("RUN.fdf", base="results/")` to make it default to the results folder. So it should already be possible to bypass? Could you play with this instead?

Reply:
> In siesta you can just do SystemLabel results/graphene and everything will be placed in the results folder. Then fdfSile will automatically handle this abstraction. So I don't think it is necessary.

I'll have to learn how to use SIESTA then :laughing: I didn't know that.

> Optionally one can initialize the fdfSile("RUN.fdf", base="results/") to make it default to the results folder. So it should already be possible to bypass? Could you play with this instead?

Yes, I will check it out, also didn't know about this.

Reply:
Ok, FYI, when using the `SystemLabel` it is important that you create the folder in advance ;)